server/app/auth/oidc/CiviformOidcLogoutActionBuilder.java

```diff
@@ -37,17 +47,29 @@ public final class CiviformOidcLogoutActionBuilder extends OidcLogoutActionBuild
 
   private String postLogoutRedirectParam;
   private final String clientId;
+  private Provider<AccountRepository> accountRepositoryProvider;
+  private final IdTokensFactory idTokensFactory;
+  private boolean isAdmin;
+  private SettingsManifest settingsManifest;
 
   public CiviformOidcLogoutActionBuilder(
-      Config civiformConfiguration, OidcConfiguration oidcConfiguration, String clientId) {
+      OidcConfiguration oidcConfiguration,
+      String clientId,
+      OidcClientProviderParams params,
+      boolean isAdmin) {
```

Comment:
We cannot use `Role`, because a TI can be authenticated by either the applicant IP or the admin IP. Revised this to use an `enum` instead of a `boolean`, which is an improvement